Topics/Tech_Stacks/Flutter.md

```diff
@@ -0,0 +1,49 @@
+# Learning Flutter
+
+## Table of contents
+
+#### [What is Flutter](#what-is-flutter)
+
+#### [Creating your first flutter app](#creating-your-first-flutter-app)
+
+## What is Flutter?
```

Comment:
Maybe you could have a section weighing the pros and cons of Flutter compared to other cross-platform frameworks like React Native? Maybe also the drawbacks compared to building native apps with Swift or Kotlin as well!